From e0d3fc7650ab35446ab9a50f58d962b26b3c1288 Mon Sep 17 00:00:00 2001
From: Allen Wild <allenwild93@gmail.com>
Date: Sat, 26 Aug 2017 02:13:25 -0400
Subject: [PATCH 2/3] bind: split shared libs into a separate bind-lib package

Anything which links to libdns or any other library from bind gets an
automatic RDEPENDS on the entire bind package, including the
auto-starting named systemd service and a bunch of other unneeded
binaries/files. Move the shared libraries to their own package to
avoid this.
---
 meta/recipes-connectivity/bind/bind_9.10.6.bb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meta/recipes-connectivity/bind/bind_9.10.6.bb b/meta/recipes-connectivity/bind/bind_9.10.6.bb
index 8b8835ba80..01ace2ea47 100644
--- a/meta/recipes-connectivity/bind/bind_9.10.6.bb
+++ b/meta/recipes-connectivity/bind/bind_9.10.6.bb
@@ -63,8 +63,9 @@ PARALLEL_MAKE = ""
 RDEPENDS_${PN} = "python3-core"
 RDEPENDS_${PN}-dev = ""
 
-PACKAGE_BEFORE_PN += "${PN}-utils"
+PACKAGE_BEFORE_PN += "${PN}-utils ${PN}-lib"
 FILES_${PN}-utils = "${bindir}/host ${bindir}/dig"
+FILES_${PN}-lib = "${libdir}/*.so.*"
 FILES_${PN}-dev += "${bindir}/isc-config.h"
 FILES_${PN} += "${sbindir}/generate-rndc-key.sh ${PYTHON_SITEPACKAGES_DIR}"
 
-- 
2.15.0

